eng/doc/fips/README.md

```diff
@@ -42,7 +42,7 @@ The Microsoft Go fork provides several ways to configure the crypto backend and
   - [`GOEXPERIMENT` `allowcryptofallback`](#build-option-to-use-go-crypto-if-the-backend-compatibility-check-fails)
   - [`import _ "crypto/tls/fipsonly"` source change](#tls-with-fips-compliant-settings)
 - Runtime configuration:
-  - [`GOFIPS` environment variable](#usage-runtime)
+  - [`GODEBUG=fips140` setting](#usage-runtime)
```

Comment:
I think GOFIPS should still be mentioned here until it's no longer in a supported version of Microsoft Go. (At least, this list was intended to include everything.)